src/stats/stats-service-openmetrics.c

```diff
@@ -141,8 +141,8 @@ openmetrics_export_metric_value(struct openmetrics_request *req, string_t *out,
 			    timestamp);
 		break;
 	case OPENMETRICS_METRIC_TYPE_DURATION:
-		str_printfa(out, " %"PRIu64" %"PRId64"\n",
-			    stats_dist_get_sum(metric->duration_stats),
+		str_printfa(out, " %f %"PRId64"\n",
+			    stats_dist_get_sum(metric->duration_stats)/1000.0,
```

Comment:
why is this being divided by 1000?

Reply:
To convert from milliseconds to seconds, which is the recommended unit for prometheus metrics. https://prometheus.io/docs/practices/naming/#base-units